pallets/pallet-bonded-coins/src/mock.rs

```diff
@@ -11,3 +50,305 @@ pub fn assert_relative_eq(target: Float, expected: Float, epsilon: Float) {
 		target
 	);
 }
+
+pub(crate) fn get_linear_bonding_curve() -> Curve<Float> {
+	let m = Float::from_num(0);
+	let n = Float::from_num(2);
+	let o = Float::from_num(3);
+	Curve::Polynomial(PolynomialParameters { m, n, o })
+}
+
+pub(crate) fn get_linear_bonding_curve_input() -> CurveInput<FloatInput> {
+	let m = FloatInput::from_num(0);
+	let n = FloatInput::from_num(2);
+	let o = FloatInput::from_num(3);
+	CurveInput::Polynomial(PolynomialParametersInput { m, n, o })
+}
+
+pub(crate) fn calculate_pool_id(currencies: Vec<AssetId>) -> AccountId {
+	AccountId::from(currencies.blake2_256())
+}
+
+#[cfg(test)]
+pub mod runtime {
+
+	use super::*;
+
+	pub type Block = frame_system::mocking::MockBlock<Test>;
+
+	pub fn generate_pool_details(
+		currencies: Vec<AssetId>,
+		curve: Curve<Float>,
+		transferable: Option<bool>,
```

Comment:
An `Option<bool>` is probably useless. Why not simply a `bool`?

Reply:
True: [575271a](https://github.com/KILTprotocol/kilt-node/pull/780/commits/575271a11853f955abae7fa29517d1be4022b437)